go.mod

```diff
@@ -1,110 +1,74 @@
-	golang.org/x/tools v0.16.1 // indirect
-	google.golang.org/appengine v1.6.7 // indirect
-	google.golang.org/genproto/googleapis/rpc v0.0.0-20230525234030-28d5490b6b19 // indirect
+	golang.org/x/tools v0.21.1-0.20240508182429-e35e4ccd0d2d // indirect
+	google.golang.org/genproto/googleapis/rpc v0.0.0-20240528184218-531527333157 // indirect
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 	k8s.io/apiextensions-apiserver v0.26.4 // indirect
-	k8s.io/kube-openapi v0.0.0-20230717233707-2695361300d9 // indirect
+	k8s.io/kube-openapi v0.0.0-20240228011516-70dd3763d340 // indirect
 	kubevirt.io/controller-lifecycle-operator-sdk/api v0.0.0-20220329064328-f3cc58c6ed90 // indirect
 	sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd // indirect
-	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
+	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
-replace (
-	github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
-	github.com/opencontainers/runc => github.com/opencontainers/runc v1.1.3
-	github.com/openshift/api => github.com/openshift/api v0.0.0-20191219222812-2987a591a72c
-	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20210112165513-ebc401615f47
-	github.com/operator-framework/operator-lifecycle-manager => github.com/operator-framework/operator-lifecycle-manager v0.0.0-20190128024246-5eb7ae5bdb7a
-	github.com/u-root/u-root => github.com/u-root/u-root v1.0.1
-	golang.org/x/text => golang.org/x/text v0.9.0
-	gopkg.in/yaml.v2 => gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api => k8s.io/api v0.28.6
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.28.6
-	k8s.io/apimachinery => k8s.io/apimachinery v0.28.6
-	k8s.io/apiserver => k8s.io/apiserver v0.28.6
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.28.6
-	k8s.io/client-go => k8s.io/client-go v0.28.6
-	k8s.io/cloud-provider => k8s.io/cloud-provider v0.28.6
-	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.28.6
-	k8s.io/code-generator => k8s.io/code-generator v0.28.6
-	k8s.io/component-base => k8s.io/component-base v0.28.6
-	k8s.io/component-helpers => k8s.io/component-helpers v0.28.6
-	k8s.io/controller-manager => k8s.io/controller-manager v0.28.6
-	k8s.io/cri-api => k8s.io/cri-api v0.28.6
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.28.6
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.28.6
-	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.28.6
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.28.6
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.28.6
-	k8s.io/kubectl => k8s.io/kubectl v0.28.6
-	k8s.io/kubelet => k8s.io/kubelet v0.28.6
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.28.6
-	k8s.io/metrics => k8s.io/metrics v0.28.6
-	k8s.io/mount-utils => k8s.io/mount-utils v0.28.6
-	k8s.io/pod-security-admission => k8s.io/pod-security-admission v0.28.6
-	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.28.6
```

Comment:
from what I remember, I think we were locking in to a specific version of the k8s and other dependencies (here in the replace stanza) to avoid situations where dependencies of dependencies results in multiple versions being vendored.